### PR TITLE
nia docs: add status api

### PR DESCRIPTION
**Pull request by lornasong**
- Configuration page: port config
 - API page covering endpoints and parameters
 - Tasks page: additional details on event, task status, overall status hierarchy

Links to changes:
 - Port config: https://deploy-preview-9362--consul-docs-preview.netlify.app/docs/nia/installation/configuration#port
 - New API page: https://deploy-preview-9362--consul-docs-preview.netlify.app/docs/nia/api
 - Task page status content: https://deploy-preview-9362--consul-docs-preview.netlify.app/docs/nia/tasks#status-information

### CODE REVIEW

File: website/pages/docs/nia/installation/configuration.mdx [hunk 1]
```diff
@@ -66,15 +68,16 @@ buffer_period {
...
   * `facility` - `(string: <none>)` Name of the syslog facility to log to.
   * `name` - `(string: "consul-terraform-sync")` Name to use for the daemon process when logging to syslog.
-* `buffer_period` - Configures the default buffer period for all [tasks](#task) to dampen the affects of flapping services to downstream network devices. It defines the minimum and maximum amount of time to wait for the cluster to reach a consistent state and accumulate changes before triggering task executions. The default is enabled to reduce the number of times downstream infrastructure is updated within a short period of time. This is useful to enable in systems that have a lot of flapping.
-  * `enabled` - `(bool: true)` Enable or disable buffer periods globally. Specifying `min` will also enable it.
-  * `min` - `(string: 5s)` The minimum period of time to wait after changes are detected before triggering related tasks.
-  * `max` - `(string: 20s)` The maximum period of time to wait after changes are detected before triggering related tasks. If `min` is set, the default period for `max` is 4 times the value of `min`.
```
Comment (lornasong) [inline website/pages/docs/nia/installation/configuration.mdx:77]: wasn't sure where to put `port` so I alphabetized this

File: website/pages/docs/nia/api.mdx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+ - Undetermined: No event data is stored for the task.
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/status/tasks/:task` | `application/json` |
+
+#### Request Parameters
+
+* `task` - `(string)` Option to specify the name of the task to return in the response. If not specified, all tasks are returned in the response.
+* `include` - `(string)` Only accepts the value "events". Use to request including stored event information with response.
```
Comment (findkim) [inline website/pages/docs/nia/api.mdx:98]: > Use to request including stored event information with response.

This is a bit difficult to read, do you mind rewording it?

File: website/pages/docs/nia/installation/configuration.mdx (hunk 1 above)
Comment (findkim) [reply website/pages/docs/nia/installation/configuration.mdx:77]: I'm pretty sure I made a mess and these aren't consistent throughout this page to begin with. Order is nice 🙂

**Review by findkim — APPROVED** [review]

File: website/pages/docs/nia/api.mdx (hunk 2 above)
Comment (lornasong) [reply website/pages/docs/nia/api.mdx:98]: :nod: what do you think about "Use to include stored event information in response."?

Comment (findkim) [reply website/pages/docs/nia/api.mdx:98]: 👍

**Comment by lornasong** [comment]
Thanks for the review @findkim! I reworded the sentence that was difficult to read and changed lists to `list[string]` syntax and changed `struct` to reference actual object or omitted. Thanks for chatting about config types!

**Pull request merged**